### PR TITLE
Update golang dependencies

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind dependency-change

#### What this PR does / why we need it:
Bump the golang dependencies to their latest versions.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
Opening https://github.com/seccomp/containers-golang/issues/28 for clarification.

File: go.mod [hunk 1]
```diff
@@ -6,20 +6,20 @@ require (
...
-	github.com/containernetworking/cni v0.7.2-0.20200304161608-4fae32b84921
+	github.com/containernetworking/cni v0.8.0
 	github.com/containernetworking/plugins v0.8.6
-	github.com/containers/buildah v1.14.9
-	github.com/containers/common v0.9.1
-	github.com/containers/conmon v2.0.16+incompatible
-	github.com/containers/image/v5 v5.4.4
-	github.com/containers/libpod v1.9.2
+	github.com/containers/buildah v1.15.0
+	github.com/containers/common v0.14.0
```
Comment (TomSweeneyRedHat) [inline go.mod:14]: is there a reason to go to 0.14 and not 0.15?

Comment (saschagrunert) [reply go.mod:14]: I think the release happened after this PR, I can update it when the times comes :)

**Comment by saschagrunert** [comment]
Let's just bump some dependencies, I'll track the other issue down further soon.

**Pull request merged**